backend/src/main/java/de/neuefische/readandmeet/backend/model/Meeting.java

```diff
@@ -0,0 +1,22 @@
+package de.neuefische.readandmeet.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.LocalDate;
+
+@Document
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Meeting {
+
+    @Id
+    private String id;
+    private String title;
+    private LocalDate date;
```

Comment:
`private String date`

Reply:
better to store the date in Mongodb as String than as LocalDate

Reply:
true, but as an Input it is easier as LocalDate 😊